.github/workflows/reusable_fast.yml

```diff
@@ -1,76 +1,58 @@
-# Fast builds
 name: FastBuild
 
 on: workflow_call
 
 permissions:
   contents: read
+  packages: read
 
 env:
   BUILD_DIR : "${{github.workspace}}/build"
   INSTL_DIR : "${{github.workspace}}/../install-dir"
 
 jobs:
-  FastBuild:
-    name: Fast builds
+  FastBuild_Linux:
+    name: Fast builds (Linux)
```

Comment:
`Linux` should be fine (we know it's a Fast builds job based on the workflow name 😉